docs/using-latest-daily.md

```diff
@@ -21,6 +21,9 @@ The latest builds are pushed to a special feed, which you need to add:
 dotnet nuget add source --name dotnet9 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json
 ```
 
+> [!TIP]
```

Comment:
I'm not sure this is true - if you don't need dotnet9-transport for regular dogfooding, you shouldn't need it for release branch dogfooding either. In fact, now that I think of it, customer-facing packages would never be on the transport feed, so you shouldn't ever need it for dogfooding.

Reply:
removed